### PR TITLE
#5046 remove unused dependencies

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5046)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-configurator/build.gradle [hunk 1]
```diff
@@ -36,11 +36,11 @@ dependencies {
   implementation libs.kafkaClient
   implementation libs.kafkaConnectFile
   implementation libs.kafkaConnectRuntime
+  implementation libs.kafkaStreams
   implementation libs.slf4jApi
   implementation libs.slf4jLog4j
   implementation libs.akkaStream
   implementation libs.akkaHttpSprayJson
-  implementation libs.rocksdb
```
Comment (jackyoh) [inline ohara-configurator/build.gradle:43]: RocksDataStore 有用到 rocksdb 的 library, 這邊移除掉會造成 compiler error 嗎？

**Comment by jackyoh** [comment]
retry connector

File: ohara-configurator/build.gradle (hunk 1 above)
Comment (chia7712) [reply ohara-configurator/build.gradle:43]: 不會喔 因為rocksdb會從kafka那邊被引入 所以我們自己定義反而可能造成多版本

Comment (jackyoh) [reply ohara-configurator/build.gradle:43]: 了解，thank you.

**Pull request merged**